pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -283,12 +296,6 @@ func (asw *actualStateOfWorld) AddVolumeNode(
 				volumeSpec.Name(),
 				err)
 		}
-	} else {
-		// volumeSpec is nil
-		// This happens only on controller startup when reading the volumes from node
-		// status; if the pods using the volume have been removed and are unreachable
-		// the volumes should be detached immediately and the spec is not needed
-		volumeName = uniqueName
```

Comment:
why don't we need this case anymore? It is for volume reconstruction when controller is restarted right?